test/infra/infra-dbdeployer-mariadb10/docker/entrypoint.sh

```
@@ -0,0 +1,151 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+echo "========================================================================"
+echo "dbdeployer entrypoint: deploying MariaDB 10.11 replication (3 nodes)..."
+echo "========================================================================"
+
+# Detect the unpacked MariaDB version
+MYSQL_VERSION=$(ls /root/opt/mysql/ | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)